lib/Driver/DependencyGraph.cpp

```diff
@@ -10,17 +10,18 @@
 //
 //===----------------------------------------------------------------------===//
 
-#include "swift/Basic/Statistic.h"
 #include "swift/Driver/DependencyGraph.h"
+#include "swift/Basic/Statistic.h"
 #include "swift/Demangling/Demangle.h"
+#include "swift/FrontendTool/ReferenceDependencies.h"
```

Comment:
The Driver library doesn't depend on FrontendTool (which you can check in the CMakeLists.txt file), so the string constants should probably live elsewhere, even though they're not going to create a link dependency. I think it's fine to make a separate header in Frontend like you did for OutputFileMap.h, and then put the old ReferenceDependencies.h back into lib/ (since those entry points aren't meant to be used from other libraries).

Reply:
Done!